src/main/com/intellij/lang/jsgraphql/ide/project/schemastatus/GraphQLSchemaEndpointsListNode.java

```diff
@@ -78,7 +78,7 @@ public ConfigurableEndpointNode(SimpleNode parent, String projectKey, GraphQLCon
             myName = endpoint.name;
             getTemplatePresentation().setTooltip("Endpoints allow you to perform GraphQL introspection, queries and mutations");
             getTemplatePresentation().setLocationString(endpoint.url);
-            setIcon(JSGraphQLIcons.UI.GraphQLNode);
+            setIcon(JSGraphQLIcons.Logos.GraphQL);
```

Comment:
I see a potential UX issue here in re-using the exact same icon for endpoints as for files (the plain GraphQL logo).

Did UX consider using a dedicated icon for the endpoints to differentiate them more clearly from files? The original version had a separate icon for endpoints for this very reason.

Reply:
Hi @jimkyndemeyer, we didn't have a lot of time to fully review all the use cases so we've decided to use the same icon right now and then have a close look at all of them after the release. If you think that it's a critical problem right now, I will ask them to prepare a new icon.

Reply:
I have the plugin up and running now with the new icons, so can easily do a quick review and make the necessary minor adjustments before the release.

Give my thanks to the UX team for the SVG files. We can always revisit later :)